testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/util/AccountHelper.java

```diff
@@ -25,14 +25,29 @@
 
 public class AccountHelper {
 
-    public static boolean updatePassword(RealmResource realm, String username, String password) {
+    public static UserRepresentation getUserRepresentation(RealmResource realm, String username) {
         Optional<UserRepresentation> userResult = realm.users().search(username).stream().findFirst();
         if (userResult.isEmpty()) {
             throw new RuntimeException("User with username " + username + " not found");
         }
 
-        UserRepresentation userRepresentation = userResult.get();
-        UserResource user = realm.users().get(userRepresentation.getId());
+        return userResult.get();
+    }
+
+    public static void logout(RealmResource realm, String username) {
+        realm.users().get(getUserRepresentation(realm, username).getId()).logout();
+    }
+
+    public static String getUsername(RealmResource realm, String username) {
+        return getUserRepresentation(realm, username).getUsername();
```

Comment:
What is the difference between the parameter `username` and the returned in `getUsername`? Isn't it the same?